src/main/java/org/broadinstitute/consent/http/models/DarCollectionSummary.java

```diff
@@ -52,6 +52,12 @@ public class DarCollectionSummary {
   @JsonProperty
   private int datasetCount;
 
+  @JsonProperty
+  private Map<String, Set<String>> parentToReferenceIds;
```

Comment:
Left the key as <String> knowing that we're moving back to using the UUID String value here.

Reply:
I actually changed my mind and made this an integer, since that's where we are right now.  This should be adjusted when we make the change.